rseqc-hg38-gencode23/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:14.04
+
+MAINTAINER John Vivian, jtvivian@gmail.com
+
+RUN apt-get update && apt-get install -y \
+            build-essential \
+            python-dev \
+            R-base \
+            samtools \
+            python-pip \
+            python-numpy
+
+RUN pip install cython
+
+# Install and setup RSeQC
+COPY RSeQC-2.6.3.tar.gz /opt/
+RUN cd /opt && tar -zxf RSeQC-2.6.3.tar.gz
+RUN cd /opt/RSeQC-2.6.3 && python setup.py install
```

Comment:
The python scripts actually get called from within that directory (see rseqc.sh, line 2). The `setup.py install` doesn't actually create any entrypoints or set anything on the PYTHONPATH, it compiles some of the included cython modules into importable .py files and checks for a couple of dependencies.